src/store/modules/session.js

```diff
@@ -70,6 +71,7 @@ const mutations = {
    * @param players
    */
   addHistory(state, players) {
+    if (!state.recordVoteHistory) return;
```

Comment:
I wondered about this too. I have mixed feelings - the ST being fallible seems part of the game, equally maybe the ST should be trusted with any support available. 

@eddgabriel or @bra1n do you have an opinion on this? (Whether an ST disabling vote history for all players should also be disabling it for themselves.)

Reply:
At the end of the day, if the ST doesn't want to use it, they don't need to open the history. I can't see a use-case where an ST actively wants to remove this functionality from themselves as well as the players and couldn't just choose to not hit V, if that makes sense?

Reply:
Ok thanks, done - since they now see it in all cases there is nothing visible to assure ST that the vote history really is hidden from players, so I've renamed the menu option from "record" to "permit".